src/test/java/HW21.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class HW21 extends BaseTest {
+
+    @Test
+     public void renamePlaylist() throws InterruptedException {
+        login("zahinedu@gmail.com", "te$t$tudent");
+        editList();
+        newName();
+//        doesPlaylistExist();
+
+    }
+
+    public void editList() {
+      wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(5)")));
+        WebElement editList = driver.findElement(By.cssSelector(".playlist:nth-child(5)"));
+       actions.doubleClick(editList).perform();
+
+    }
+   public  void newName() {
+        WebElement playlistInputField= wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("input[name='name']")))
+;
+       playlistInputField.sendKeys((Keys.chord(Keys.CONTROL,"a",Keys.BACK_SPACE)));
+       playlistInputField.sendKeys("HOMEWORK21");
+       playlistInputField.sendKeys(Keys.ENTER);
+   }
+}
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"

The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._